packages/components/src/ConfirmationModal/ConfirmationModal.mdx

```diff
@@ -40,6 +40,7 @@ import {
           onConfirm={() => alert("✅")}
```

Comment:
I'd like to change this sandbox example up a bit:

```
() => {
  const [open, setOpen] = useState(false)
  return (
    <>
      <Button type="secondary" variation="destructive" label="Delete Bob" onClick={() => setOpen(true)} />
      <ConfirmationModal
        title="Delete Bob?"
        message={`Deleting Bob will remove their data from your account for good.`}
        confirmLabel="Delete Bob"
        open={open}
        onConfirm={() => alert('Bob has been deleted')}
        onCancel={() => alert('Bob will not be deleted')}
        onRequestClose={() => setOpen(false)}
        type="destructive"
      />
    </>
  )
}
```

As I think it's a clearer demonstration of the use case this way

Reply:
Sure thing !